### PR TITLE
Stop installing or writing files in user space per CRAN rules

### PR DESCRIPTION
**Pull request by ddl-gabrielhaim**
Fix issues with CRAN rules

https://dominodatalab.atlassian.net/browse/DOM-47920

### CODE REVIEW

File: man/py_select_interpreter.Rd [hunk 1]
```diff
@@ -0,0 +1,11 @@
+% Generated by roxygen2: do not edit by hand
```
Comment (ddl-joyce-zhao) [inline man/py_select_interpreter.Rd:1]: qq: how is this get generated? What command to run? or it's in the pre-commit hook?

**Review by ddl-joyce-zhao — APPROVED** [review]
LGTM

File: man/py_select_interpreter.Rd (hunk 1 above)
Comment (ddl-gabrielhaim) [reply man/py_select_interpreter.Rd:1]: This works but probably should be in docs or automated: `R --no-save -e "devtools::document()"`

Comment (ddl-joyce-zhao) [reply man/py_select_interpreter.Rd:1]: Thank you! Probably it can be added to README for whoever is not familiar with this generation.

**Pull request merged**